pkg/virt-handler/vm.go

```diff
@@ -2385,6 +2394,22 @@ func (d *VirtualMachineController) vmUpdateHelperMigrationSource(origVMI *v1.Vir
 			d.recorder.Event(vmi, k8sv1.EventTypeNormal, v1.Migrating.String(), "VirtualMachineInstance is aborting migration.")
 		}
 	} else {
+		if isMigrationInProgress(vmi, domain) {
```

Comment:
I think we could extend `handleSourceMigrationProxy` to know if the proxies are running or not and build a better picture from it. Something to consider for the future.